helm/dashboards/charts/private_dashboards_mz/dashboards/shared/private/prometheus-mimir-comparative.json

```diff
@@ -94,7 +94,7 @@
               "uid": "PBFA97CFB590B2093"
             },
             "editorMode": "code",
-            "expr": "sum(rate(container_cpu_usage_seconds_total{container=\"prometheus\", pod=~\"prometheus-($cluster)-.*\", cluster_type=~\"management_cluster\"}[5m]))",
+            "expr": "sum(rate(container_cpu_usage_seconds_total{container=\"prometheus\", namespace=~\".*-prometheus\", cluster_type=~\"management_cluster\"}[$__range]))",
```

Comment:
We use the namespace instead of the live prometheus as this condition is missing a lot of historical data especially on test MCs where clusters are created/deleted multiple times a day